### DISCUSSION TITLE
GitHub Support does not respond

### DISCUSSION THREAD

**Discussion by RamitaDeeprom** [community/community#138541] [Other Feature Feedback, Questions, & Ideas]
Hi everyone,

I’m trying to help my friend (**@buritbest321**) regain access to their GitHub account after receiving a **suspicious activity notification**. Despite following the **password reset instructions**, they’re unable to log in. Additionally, their **GitHub account** is no longer visible when searched. We’ve reached out to **GitHub Support** with **Ticket ID: 2994089**, but haven’t received a response yet.

Has anyone else experienced similar issues or have advice on how to expedite the process with **GitHub Support**? Any help would be greatly appreciated.

Thank you!

![image](https://github.com/user-attachments/assets/bbf7a71b-7edd-4d49-bfd6-0f3bd9a9250f)
![image](https://github.com/user-attachments/assets/e3190434-f89c-46ee-9f7b-245f88dd7cb1)

https://github.com/user-attachments/assets/95a54cf7-c6a7-4d30-a739-6f0d89273d9f

**Comment by QuackerOatmeal** [comment]
Same issue here. I've been marked as spam and unable to download anything for CS50p. Not even sure this comment will be allowed to be posted. It's been two weeks since I submitted a ticket and I haven't heard a word back.

**Comment by jovanovic-djo** [comment]
As I can see, you entered different email address from the one they have tried to verify. Isn't verification for buritbest321@gmail.com instead of bestburit32121@gmail.com? This leads to my conclusion that you are not logged in the right e mail address or you did not entered right one.

**Reply by buritbest123** [reply]
It's not the email address that I use for verification. My GitHub ID is @buritbest321, but the email address I registered with GitHub is bestburit32121@gmail.com. So, the verification should be done using that email. The screenshot shows that the email address is invalid.

Could you please assist further with this issue?
![image](https://github.com/user-attachments/assets/4d3d84b7-33bb-4541-98e9-7d9ca73c2c83)

**Reply by jovanovic-djo** [reply]
The process of verifying an account is intuitive and well known for the most of the platforms and services, I hope that you are not doing it in the wrong way, please double check if you are doing things right. Otherwise, there is not much that can be done by any individual here. 

If the issue is on your side, I am sure that you will be able to resolve it anytime soon, but if you think that there is an problem on their side, I would suggest you to contact support at: https://support.github.com/ , whey will be able to help you if you conclude that there is uncommon issue.

**Comment by ghostinhershell** [comment]
Hi @RamitaDeeprom,

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
